### ISSUE TITLE
julianDayUT and julianDayET are mixed up in swe_utc_to_jd results

### ISSUE THREAD

**Issue by dtamas** [mivion/swisseph#19]
```javascript
var swisseph = require('swisseph');

var jd = swisseph.swe_utc_to_jd(2015, 10, 18, 12, 0, 0, 0, swisseph.SE_GREG_CAL);
var ut = swisseph.swe_jdut1_to_utc(jd.julianDayUT, swisseph.SE_GREG_CAL);
console.log(ut.second == 0);
var et = swisseph.swe_jdet_to_utc(jd.julianDayET, swisseph.SE_GREG_CAL);
console.log(et.second == 0);
```

**Comment by mivion** [comment]
Hi,

These result are returned by swisseph library, see the documentation for this functions:
http://www.astro.com/swisseph/swephprg.htm

Also to get julian date use function swe_julday ().

**Comment by dtamas** [comment]
...in more detail:  please compare 

https://github.com/mivion/swisseph/blob/master/deps/swisseph/swedate.c#L471
```
dret[0] = tjd_et;
dret[1] = tjd_ut1;
```

with 

https://github.com/mivion/swisseph/blob/master/src/date.cc#L180
```
result->Set (Nan::New<String> ("julianDayUT").ToLocalChecked(), Nan::New<Number> (tjd [0]));
result->Set (Nan::New<String> ("julianDayET").ToLocalChecked(), Nan::New<Number> (tjd [1]));
```

please note, swedate.c returns an array [ET, UT],  while date.cc assumes [UT, ET]

**Comment by knyazhe1** [comment]
```javascript
//deps/swisseph/swedate.c:
/*
int32 FAR PASCAL_CONV swe_utc_to_jd(int32 iyear, int32 imonth, int32 iday, int32 ihour, int32 imin, double dsec, int32 gregflag, double *dret, char *serr)
...
  dret[0] = tjd_et;
  dret[1] = tjd_ut1;
*/
//src/date.cc
/*
* int32 swe_utc_to_jd(int32 iyear, int32 imonth, int32 iday, int32 ihour, int32 imin, double dsec, int32 gregflag, double *dret, char *serr)
* =>
* swe_utc_to_jd(int32 iyear, int32 imonth, int32 iday, int32 ihour, int32 imin, double dsec, int32 gregflag) {
*   julianDayUT: double,
*   julianDayET: double,
...
		result->Set (Nan::New<String> ("julianDayUT").ToLocalChecked(), Nan::New<Number> (tjd [0]));
		result->Set (Nan::New<String> ("julianDayET").ToLocalChecked(), Nan::New<Number> (tjd [1]));
*/
var assert = require ('assert');
var swisseph = require ('..');

var version = swisseph.swe_version ();

console.log ('Swiss Ephemeris version:', version);

// Test date
var date = new Date(); // !!! javascript Date 
console.log ('Test date:', date, date.getMilliseconds(), 'ms');

var flag = swisseph.SEFLG_SPEED | swisseph.SEFLG_MOSEPH;

// path to ephemeris data
swisseph.swe_set_ephe_path (__dirname + '/../ephe');

// Time zone
var t;
swisseph.swe_utc_time_zone (
	date.getFullYear(), 
	date.getMonth(), 
	date.getDate(), 
	date.getHours(), 
	date.getMinutes(), 
	date.getSeconds() + date.getMilliseconds() / 1000, 
	5.5, 
	function (result) {t = result;}
	);
console.log('--- swe_utc_time_zone:'); 
console.log(t);

// Julian UT|ET days
swisseph.swe_utc_to_jd (
	t.year, 
	t.month, 
	t.day, 
	t.hour, 
	t.minute, 
	t.second, 
	swisseph.SE_GREG_CAL, 
	function (result) {
		assert (!result.error, result.error);
		t = result.julianDayET;
		// !!! uncomment next line for right result
		//t = result.julianDayUT;
	});
console.log('--- swe_utc_to_jd:');
console.log(t);

// Reverse date conversion from UT
swisseph.swe_jdet_to_utc (
	t, 
	swisseph.SE_GREG_CAL, 
	function (result) {t = result;}
	);
console.log('--- swe_jdet_to_utc:');
console.log(t);

swisseph.swe_utc_time_zone (
	t.year, 
	t.month, 
	t.day, 
	t.hour, 
	t.minute, 
	t.second, 
	-5.5, 
	function (result) {t = result;}
	);
console.log('--- swe_utc_time_zone:');
console.log(t);

var intS = Math.round(t.second);
if(intS > t.second)
	intS = intS - 1;

var mS = Math.round(1000 * (t.second - intS));

var d = new Date(t.year, t.month, t.day, t.hour, t.minute, intS, mS);
console.log('Finished:', d, d.getMilliseconds(), 'ms');
assert.equal(d.getTime(), date.getTime());
```

**Comment by mivion** [comment]
Hi,

Thanks for reply.
Can you create PR to fix this?

**Comment by knyazhe1** [comment]
ok, i will try it

**Comment by mivion** [comment]
Hi,

Merged and released version 0.5.7.

**Comment by mivion** [comment]
Please checkout and test the issue.
